erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-21-report.md

```diff
@@ -0,0 +1,21 @@
+# UAE VAT 21 Report in ERPNext
+
+To generate GSTR3B Report in ERPNext navigate to <br>
+**Accounting > Value-Added Tax (VAT UAE) > UAE VAT 21** <br>
+or simply search for UAE VAT 21 in awesomebar.
+
+<img class="screenshot" alt="UAE VAT 21 Report" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-21-report.png">
+
+If you want to print the report it can also be printed and viewed in by clicking on Print in Menu as shown below.
+
+<img class="screenshot" alt="UAE VAT 21 Report Download" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-21-download.png">
+
+Note: To make sure the report is calculated accurately and correctly please make sure of the following things.
+
+1. Correct Emirate is selected for each Address.
+
+2. Nil rated or exempted Items have Is nil rated or Is Exempt checkbox checked in the item master.
+
+3. ALL VAT Accounts are selected in UAE VAT Settings.
+
+<img class="screenshot" alt="UAE VAT Account Settings" src="{{docs_base_url}}/assets/img/regional/uae/uae-vat-account-settings.png">
```

Comment:
crop the comment part (below part of the image). Take the screenshot only till add row button